lm_eval/tasks/basque_bench/flores_eu/create-yamls_flores_eu.py

```diff
@@ -0,0 +1,115 @@
+"""
+Script to generate task YAMLs for the FLORES-200 dataset.
+Based on `tasks/translation/utils.py`.
```

Comment:
Could we remove the hyphen in these yaml-generation scripts? **Hyphens are not allowed in python filenames.**